vcpkg/consume/asset-caching.md

```diff
@@ -82,8 +84,8 @@ set "X_VCPKG_ASSET_SOURCES=clear;x-azurl,file://D:/vcpkg/asset-cache,,readwrite"
 This `X_VCPKG_ASSET_SOURCES` configuration adds the following source strings:
 
 * `clear`, disables any previously configured asset cache
-* `x-azurl,file://C:/vcpkg/asset-cache,,readwrite`, sets a filesystem asset cache, located in
-  `D:\vcpkg\asset-cache`, with read-write permissions.
+* `x-azurl,file:///Z:/vcpkg/asset-cache,,readwrite`, sets a filesystem asset cache, located in
```

Comment:
Is changing everything to Z: a convention we want to use everywhere? If not I think it would be better to change less by not changing these.

Reply:
`Z:` is used in other tutorials, I like it because it is a custom location that is unlikely to exist for most users.